squire_web/src/tournament/overview/mod.rs

```diff
@@ -27,16 +27,32 @@ impl Component for TournOverview {
             .unwrap()
             .state
             .query_tournament(&self.id, |t| {
+                let activeRounds = t.round_reg.rounds.values().filter(|r| r.is_active()).count();
```

Comment:
While this works, you are passing through all the rounds twice when this can be done in a single pass:
```suggestion
                let (active, certified) = t.round_reg.rounds.values().fold((0, 0), |acc, r| {
                     match r.status {
                         RoundStatus::Open => acc.0 += 1,
                         RoundStatus::Certified => acc.1 += 1,
                          _ => {}
                     }
                     acc
                 });
```